shared/extra-utils/server/config.ts

```diff
@@ -98,7 +98,8 @@ function updateCustomSubConfig (url: string, token: string, newConfig: DeepParti
     signup: {
       enabled: false,
       limit: 5,
-      requiresEmailVerification: false
+      requiresEmailVerification: false,
+      minimumAge: 16
```

Comment:
Please use another value and check it is correctly set with the default in https://github.com/Chocobozzz/PeerTube/blob/develop/server/tests/api/server/config.ts#L63 and correctly updated in https://github.com/Chocobozzz/PeerTube/blob/develop/server/tests/api/server/config.ts#L153